### ISSUE TITLE
Inline strings are not written into spreadsheet.

### ISSUE THREAD

**Issue by PowerGamer1** [PHPOffice/PhpSpreadsheet#353]
### What is the expected behavior?
The generated spreadsheet contains text 'inline str' in A1 cell. The text appears directly in the cell inside `sheet1.xml` (and not in `sharedStrings.xml`).

### What is the current behavior?
A1 cell of generated spreadsheet is empty.

### What are the steps to reproduce?
```PHP
<?php
require __DIR__ . '/vendor/autoload.php';
$doc = new \PhpOffice\PhpSpreadsheet\Spreadsheet();
$sheet = $doc->getSheet(0);
$sheet->getCellByColumnAndRow(1, 1)->setValueExplicit('inline str', \PhpOffice\PhpSpreadsheet\Cell\DataType::TYPE_INLINE);
$sheet->getCellByColumnAndRow(2, 1)->setValueExplicit('shared str', \PhpOffice\PhpSpreadsheet\Cell\DataType::TYPE_STRING);
$writer = \PhpOffice\PhpSpreadsheet\IOFactory::createWriter($doc, 'Xlsx');
$writer->save('1.xlsx');
```
### Which versions of PhpSpreadsheet and PHP are affected?
1.0.0.

When a cell value is set with a `DataType::TYPE_INLINE` the writer should write such string into the spreadsheet as inline string (which goes directly into the content of a cell rather than shared string table). See 18.3.1.4 and 18.18.11 in ISO/IEC 29500-1 for more info.

**Comment by PowerKiKi** [comment]
I can confirm what you described. But what is the use-case for using inline strings instead of shared one ?

**Comment by PowerGamer1** [comment]
1. According to spec inline string MUST be placed into the cell itself, not string table, so as long as your library allows setting string type as inline it must adhere to spec when writing such strings into XML - and use case does not matter.

2. The whole point of shared string table is to save memory for string storage in case your worksheet has many instances of the same string in different cells. Putting there a string that appears in a single cell only and never repeats in any other cell would not only be pointless but also unnecessary increase shared string table size which might increase the lookup speed in shared string table.

**Comment by PowerKiKi** [comment]
That seems fair. If this is important to you, please consider contributing a PR for it. In the meantime we can probably workaround it by using shared strings.

**Comment by eltharin** [comment]
Hi, 
It's possible to want to open files using inline strings and for the moment the return is null.

**Comment by PowerKiKi** [comment]
@eltharin this would be a different issue, please consider opening a new issue, or better yet contributing a patch for it.

**Comment by oleibman** [comment]
Fixed by PR #2569.
